src/components/footer/Footer.tsx

```diff
@@ -1,12 +1,37 @@
 import IconLink from "./assets/external-link.svg";
 import LogoGouv from ":/assets/logo-gouv.svg";
-import { useCunningham } from "@openfun/cunningham-react";
 
+export type FooterProps = {
+  externalLinks?: {
+    label: string;
+    href: string;
+  }[];
+  legalLinks?: {
+    label: string;
+    href: string;
+  }[];
+  license?: {
+    label: string;
+    link: {
+      label: string;
+      href: string;
+    };
+  };
+  logo?: {
+    src: string;
+    width?: string;
+    height?: string;
+    alt: string;
+  };
+}
```

Comment:
Maybe we could factorize a little bit.

```suggestion
type Link {
    label: string;
    href: string;
};
export type FooterProps = {
  externalLinks?: readonly Link[]
  legalLinks?: readonly Link[];
  license?: {
    label: string;
    link: Link;
  };
  logo?: {
    src: string;
    width?: string;
    height?: string;
    alt: string;
  };
}
```